MapboxDirections/MBVoiceInstruction.swift

```diff
@@ -0,0 +1,32 @@
+import Foundation
+
+@objc(MBVoiceInstruction)
+public class VoiceInstruction: NSObject, NSSecureCoding {
+    
+    public let distanceAlongGeometry: CLLocationDirection
+
+    public let textAnnouncement: String
+    
+    public let SSMLAnnouncement: String
```

Comment:
Instruction and announcement are sort of synonymous, so having them both in this class is a bit redundant. Let’s call these methods `text` and `ssmlText`. `ssmlText` should have an Objective-C name of `SSMLText`.